action_for_github_server/action.yml

```diff
@@ -53,20 +57,72 @@ runs:
           echo "DA_ADMIN_PASSWORD=$DA_ADMIN_PASSWORD" >> "$GITHUB_OUTPUT"
         shell: bash
 
-      - name: "ALKiln github_server: Download and start docker"
+      - name: "ALKiln github_server - Get config"
         run: |
+          # Info: config values
```

Comment:
It turns out that this is what shows up in the job's console logs, not the name. Go figure.